python/redcanary/detections.py

```diff
@@ -57,15 +72,64 @@ def all(self, per_page: int = None, since: str = None) -> list:
         """
         return [Detection(i) for i in self._get_all('detections', locals())]
 
-    def byid(self, id: int) -> object:
+    def get_by_id(self, id: int) -> object:
```

Comment:
```suggestion
    def by_id(self, id: int) -> object:
```
I'd suggest somewhere in the middle ;)
The `get_` seems superfluous. Just my opinion...

Reply:
I'm used to PowerShell where everything starts with a verb